tsconfig.json

```diff
@@ -55,7 +55,7 @@
     // "sourceMap": true,                                /* Create source map files for emitted JavaScript files. */
     // "inlineSourceMap": true,                          /* Include sourcemap files inside the emitted JavaScript. */
     // "outFile": "./",                                  /* Specify a file that bundles all outputs into one JavaScript file. If 'declaration' is true, also designates a file that bundles all .d.ts output. */
-    "outDir": "./dist/src/" /* Specify an output folder for all emitted files. */,
+    "outDir": "./dist/" /* Specify an output folder for all emitted files. */,
```

Comment:
Please, check if it does not break any build scripts including CI

Reply:
It used to build in one folder, due to the fact that it is now importing from package.json, it now creates the src folder automatically. In the build everything is the same as it was, but now package.json is added.